README.md

```diff
@@ -56,7 +56,7 @@ conan install . --output-folder build --build=missing -s build_type=Release -s c
 ```
 
 ```
-.venv/bin/cmake --build build -j4  
+sudo make install -j4 -C build
```

Comment:
```suggestion
.venv/bin/cmake --build build -j4

sudo .venv/bin/cmake --install build
```

Let's keep both of them since some users might not want to install globally.